src/score_hv/harvesters/daily_bfg.py

```diff
@@ -156,41 +160,68 @@ def get_data(self):
         else:
             median_cftime = cftime.num2date(np.median(temporal_endpoints),
                                             'hours since 1951-01-01 00:00:00')
-        
+
         for i, variable in enumerate(self.config.get_variables()):
-            """ The first nested loop iterates through each requested variable
+            """ The first nested loop iterates through each requested variable.
             """
-            variable_data = xr_dataset[variable]
-            temporal_means = np.ma.masked_invalid(variable_data.mean(dim='time',
-                                                                   skipna=True))
-            if 'long_name' in variable_data.attrs:
-                longname = variable_data.attrs['long_name']
-            else:
-                longname = "None"
-            if 'units' in variable_data.attrs:
-                units = variable_data.attrs['units']
+            if variable == 'netrf_avetoa': 
+                """The variable name netrf_avetoa referes to the top of the 
+                atmosphere (TOA) net radiative energy flux. Here, functions 
+                from the stats_utils.py file are called to calculate statistics 
+                for the required component variables. The TOA net energy flux
+                is defined as the following:
+                
+                netrf_avetoa = dswrf_avetoa - uswrf_avetoa - ulwrf_avetoa,
+                
+                where dswrf_avetoa, uswrf_avetoa, ulwrf_avetoa are the
+                corresponding mean TOA downward shortwave, upward shortwave and
+                upward longwave radiation fluxes, respectively.
+                """
+                longname = "Top of atmosphere net radiative energy flux"
+                units = "W/m**2"
+                
+                #TODO: probably should add try/except block here for cases
+                # when component variables are not found
```

Comment:
note to self and/or @jrknezha to help develop exception handling for if/when component variables are not in the bfg files (or named different) for whatever reason